app/helpers/create_account.py

```diff
@@ -53,7 +53,8 @@ def create_account(
   responseObject = {
     'status': 'success',
     'message': 'Successfully registered.',
-    'auth_token': auth_token
+    'auth_token': auth_token,
+    'refresh-token': refresh_token
```

Comment:
can you make it an underscore `_` instead of dash `-` for consistency purposes

Reply:
Sure.